types/scrypt-async/scrypt-async-tests.ts

```diff
@@ -3,21 +3,55 @@ import scrypt = require("scrypt-async");
 var callback = function(key: string | number[]) { };
 
 scrypt("abc", "def", 10, 8, 32, 1000, callback, "base64");
-scrypt("abc", [4,5,6], 10, 8, 32, 1000, callback, "base64");
-scrypt([1,2,3], "def", 10, 8, 32, 1000, callback, "base64");
-scrypt([1,2,3], [4,5,6], 10, 8, 32, 1000, callback, "base64");
+scrypt("abc", [100, 101, 102], 10, 8, 32, 1000, callback, "base64");
+scrypt([97, 98, 99], "def", 10, 8, 32, 1000, callback, "base64");
+scrypt([97, 98, 99], [100, 101, 102], 10, 8, 32, 1000, callback, "base64");
 
 scrypt("abc", "def", 10, 8, 32, 1000, callback);
-scrypt("abc", [4,5,6], 10, 8, 32, 1000, callback);
-scrypt([1,2,3], "def", 10, 8, 32, 1000, callback);
-scrypt([1,2,3], [4,5,6], 10, 8, 32, 1000, callback);
+scrypt("abc", [100, 101, 102], 10, 8, 32, 1000, callback);
+scrypt([97, 98, 99], "def", 10, 8, 32, 1000, callback);
+scrypt([97, 98, 99], [100, 101, 102], 10, 8, 32, 1000, callback);
 
 scrypt("abc", "def", 10, 8, 32, callback, "base64");
-scrypt("abc", [4,5,6], 10, 8, 32, callback, "base64");
-scrypt([1,2,3], "def", 10, 8, 32, callback, "base64");
-scrypt([1,2,3], [4,5,6], 10, 8, 32, callback, "base64");
+scrypt("abc", [100, 101, 102], 10, 8, 32, callback, "base64");
+scrypt([97, 98, 99], "def", 10, 8, 32, callback, "base64");
+scrypt([97, 98, 99], [100, 101, 102], 10, 8, 32, callback, "base64");
 
 scrypt("abc", "def", 10, 8, 32, callback);
-scrypt("abc", [4,5,6], 10, 8, 32, callback);
-scrypt([1,2,3], "def", 10, 8, 32, callback);
-scrypt([1,2,3], [4,5,6], 10, 8, 32, callback);
\ No newline at end of file
+scrypt("abc", [100, 101, 102], 10, 8, 32, callback);
+scrypt([97, 98, 99], "def", 10, 8, 32, callback);
+scrypt([97, 98, 99], [100, 101, 102], 10, 8, 32, callback);
+
+// Tests by: Stefano Sicco <https://github.com/stesix>
+// "abc" === [97, 98, 99]
+// "def" === [100, 101, 102]
+
+interface IOptions {
```

Comment:
You don't need to redeclare this -- just change `declare namespace ScryptAsync` to `declare namespace scrypt` and it will be merged with the export, then you can access `scrypt.Options` as a type.